grammar.js

```diff
@@ -227,12 +228,15 @@ module.exports = grammar({
 
     expression_statement: $ => choice(
       $.expression,
-      seq(commaSep1($.expression), optional(',')),
+      $.comma_separated_expressions,
       $.assignment,
       $.augmented_assignment,
       $.yield,
     ),
 
+    comma_separated_expressions: $ => 
```

Comment:
Maybe we can call it `tuple_expression`? This is what Python's `ast` modules gives for `1, 2, 3`

<img width="740" height="203" alt="Image" src="https://github.com/user-attachments/assets/9c7b41b6-0940-49a2-a889-a7f33b481748" />

Reply:
makes sense!